### PR TITLE
Add an About page

### PR DESCRIPTION
**Pull request by subsymbolic**
Reviewer: Caine
Asana: https://app.asana.com/0/361428290920652/375790519910153

**Description**:
Adds an about page to our app

**Steps to test this PR**:
Visit about page

###### Reviewer Checklist:
- [x] **Ensure the PR solves the problem**
- [x] **Review every line of code**
- [x] **Ensure the PR does no harm by testing the changes thoroughly**
- [x] **Get help if you're uncomfortable with any of the above!**
- [ ] Determine if there are any quick wins that improve the implementation

###### PR DRI Checklist:
- [ ] Get advice or leverage existing code
- [ ] Agree on technical approach with reviewer (if the changes are nuanced)
- [ ] Ensure that there is a testing strategy (and documented non-automated tests)
- [ ] Ensure there is a documented monitoring strategy (if necessary)
- [ ] Consider systems implications (Database connections, Grafana stats, CPU)

### CODE REVIEW

File: Core/AppDeepLinks.swift [hunk 1]
```diff
@@ -26,6 +26,8 @@ public struct AppDeepLinks {
     
     public static let quickLink = "ddgQuickLink://"
     
+    public static let privacyPolicy = URL(string: "\(AppDeepLinks.quickLink)https://duckduckgo.com/privacy")!
```
Comment (nilnilnil) [inline Core/AppDeepLinks.swift]: Please explain this? I don't get the AppDeepLinks part.

File: Core/AppDeepLinks.swift [hunk 2]
```diff
@@ -39,4 +41,8 @@ public struct AppDeepLinks {
         }
         return false
     }
+    
+    public static func query(fromQuickLink url: URL) -> String {
+        return url.absoluteString.replacingOccurrences(of: quickLink, with: "")
```
Comment (nilnilnil) [inline Core/AppDeepLinks.swift]: Is this a bookmarksl ink? What is Quick about it?

**Comment by nilnilnil** [comment]
Merge if you'd like. Mostly questioining that name.

File: Core/AppDeepLinks.swift (hunk 2 above)
Comment (subsymbolic) [inline Core/AppDeepLinks.swift]: Yeah I had trouble naming this one! It's a link that launches straight into the browser for example an external link (bookmark/favourite) or an internal link outside the browsing context (as in this case where we launch the privacy url from settings).

**Comment by subsymbolic** [comment]
Ok merging now but happy to change that name... Let me know if you think of a better one.

**Pull request merged**